location_spliter/location_spliter.py

```diff
@@ -0,0 +1,58 @@
+"""This script separates the locations into chunks for mapping to AWS Lambdas."""
+
+from os import environ as ENV
+from json import dumps
```

Comment:
Dan told us to be critical so I'm going to request that this is in alphabetical order